src/middleware.ts

```diff
@@ -26,8 +26,7 @@ function mergeActionHandlers<C, S>(
   const allHandlers: ActionHandlerSet<C, S> = {}
 
   handlers.forEach(group => {
```

Comment:
This is where we look for which callbacks to call, using the object/map mentioned above. So we change the int key lookup to a string.